kubernetes-elasticsearch-cluster/es-data-stateful.yaml

```diff
@@ -20,6 +20,8 @@ spec:
         component: elasticsearch
         role: data
     spec:
+      nodeSelector:
```

Comment:
Let's add [Pod anti-affinity](https://github.com/pires/kubernetes-elasticsearch-cluster#pod-anti-affinity) to be safe. It probably won't make a difference, but it doesn't hurt to have it. (This also stems from the baseline-baseline-explorer issues.)

Reply:
Done. I actually think this may have helped/solved the unschedulable issue, though I'm not quite sure why (haven't seen it since).